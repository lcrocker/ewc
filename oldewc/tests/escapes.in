ab~cd~~ef~~~gh~~~~ij
kl~-mn~--op~---qr~
st\
uv\wx  
yz~\
ab
ab\\
cd	
ef\\gh
