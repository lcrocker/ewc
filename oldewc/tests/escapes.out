u'ab\uef63d\uef7eef\uef7e\uef67h\uef7e\uef7eij'
u'kl\uef2d\uef6dn\uef2d\uef2d\uef6fp\uef2d\uef2d\uef2d\uef71r\xa0'
u'stuv\\wx'
u'yz\uef5c'
u'ab'
u'ab\\\\'
u'cd'
u'ef\\\\gh'
